CONTRIBUTING.md

```diff
@@ -33,7 +31,7 @@ Then you need `git`, if you don't have that yet: https://git-scm.com/
 
 And for the final step before we actually get started, you'll need build tools to install
 the native modules used by the application. On Windows, it's easiest to open the [Command
-Prompt (`cmd.exe`) as Administrator](https://technet.microsoft.com/en-us/library/cc947813(v=ws.10).aspx)
+Prompt (`cmd.exe`) as Administrator](<https://technet.microsoft.com/en-us/library/cc947813(v=ws.10).aspx>)
```

Comment:
Do you know what the reasoning is behind this change? I guess my Markdown-Fu is not all the way up 😜

Reply:
My guess is that it's because the URL itself has `(` and `)` in it.

Reply:
Ah! That’s neat!